README.md

```diff
@@ -9,6 +9,12 @@ low level abstraction for the Animated library API to be built
 on top of and hence allow for much greater flexibility especially when it
 comes to gesture based interactions.
 
+### Nightly CI state
+
+|CI state|
+|---|
```

Comment:
There's no need to duplicate "CI state"
```suggestion
```

Reply:
GitHub markdown doesn't support tables without header, but if you want I can replace it with an HTML table.

Reply:
Do we even need a table here? We can just put these two images next to each other.